src/components/Header/Navbar.jsx

```diff
@@ -7,26 +7,37 @@ import searchBtn from "../../assets/search-btn.svg";
 import { useDispatch } from "react-redux";
 // action creator for reset header poster
```

Comment:
I would move it to where it's used, but do we need this comment ? please explain

Reply:
We dont need. Deleted